src/components/views/location/ShareType.tsx

```diff
@@ -51,7 +51,6 @@ type ShareTypeOptionProps = HTMLAttributes<Element> & { label: string, shareType
 const ShareTypeOption: React.FC<ShareTypeOptionProps> = ({
     onClick, label, shareType, ...rest
 }) => <AccessibleButton
-    element='button'
```

Comment:
@weeman1337 I see the logic, so would you letting me know what your thought would be on a possible task to fix the root cause by changing `AccessibleButton` to use `button` HTML element by default?

This could be a large task but this should be logical.

If that makes sense for you too, I am going to create a ticket to track the task.

Reply:
Yes, I would say lets target that in a separate task. You could check [the issue list](https://github.com/vector-im/element-web/issues) before to prevent a duplicate.